### PR TITLE
[Component with its own class name as service type ] LPS-199464 Refactor Helper class from document-library and portal

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by GabrielHenriquedev** [comment]
ci:test:lima

**Comment by vicnate5** [comment]
No regressions found.

**Review by rafaprax — APPROVED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/CacheControlFileEntryHttpHeaderCustomizer.java [hunk 1]
```diff
@@ -47,12 +62,39 @@ public String getHttpHeaderValue(FileEntry fileEntry, String currentValue) {
...
 
+	@Activate
+	protected void activate(
+		BundleContext bundleContext, Map<String, Object> properties) {
+
+		modified(properties);
+
+		_serviceRegistration = bundleContext.registerService(
+			ManagedServiceFactory.class,
+			new CacheControlFileEntryHttpHeaderCustomizer.
```
Comment (tinatian) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/CacheControlFileEntryHttpHeaderCustomizer.java:73]: @GabrielHenriquedev Please remove `CacheControlFileEntryHttpHeaderCustomizer.`

**Comment by tinatian** [comment]
@GabrielHenriquedev @rafaprax modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/configuration/helper/CacheControlConfigurationHelper.java was added by https://github.com/liferay/liferay-portal/commit/ceaf25464c14b7c4d74453c3dc979160a81e19e6 to access company config.

**Pull request closed without merging**